server/backend/backend/settings.py

```
@@ -96,16 +97,29 @@
+        }
+    }
+else:
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'NAME': config('DB_NAME'),
+            'USER': config('DB_USER'),
+            'PASSWORD': config('DB_PASSWORD'),
+            'HOST': config('DB_HOST'),
+            'PORT': config('DB_PORT', cast=int),
+        }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)